flying-saucer-core/src/test/resources/letter-spacing-astral.html

```
@@ -0,0 +1,8 @@
+<html lang="en">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)